### ISSUE TITLE
Vote verification doesn't work

### ISSUE THREAD

**Issue by falzard** [Dawnbrightsu/FusionCMS#5]
Title says it all. It doesn't work.

I have another module which doesn't work either (vote system v2).

Someone has it working?

**Comment by Dawnbrightsu** [comment]
#Fixed!

**Comment by falzard** [comment]
Really? your commits are not descriptive :D 
You should separate them by feature, and not just by "readme update" for readability.

Would you mind showing the commit (even if it's a big commit) where you fix donation and vote?

**Comment by Dawnbrightsu** [comment]
It was fixed on the CMS version update and I am not here to give detailed commits of were I fixed stuff or stuff got fixed, because if you are using my FusionCMS you only need to download the new rev and that's it. The voting system was completely reworked on CMS update 6.1.9

**Comment by falzard** [comment]
In fact, I was wondering this because the fusionCMS nulled version that most people use was already version 6.1.21 and you updated it only recently. I don't know, I may be confused but I wonder if it's your fix or if it's the same version as the one I used (which doesn't work). I also modified lots of things for my own personal use and I don't remember what exactly (minor things, mostly design because I never hoped for anyone to fix fusionCMS so I didn't care about doing it the right way lol) so using your version may break all these little things I did.
